src/main/python/asdf_to_json/main.py

```diff
@@ -0,0 +1,77 @@
+import json
+import os.path
+import sys
+import argparse
+
+from modelforge.model import Model
+from modelforge.models import register_model
+from sourced.ml.models import OrderedDocumentFrequencies
+
+
+@register_model
+class WeightedMinHashParameters(Model):
```

Comment:
For maintainability and future contributors - if a substantial body of code comes from another project, it's better to past a permalink here i.e in comments. 

It's easy to get such link i.e like in https://github.com/src-d/gemini/commit/6a1993492f49f47e83c211d2d7bf6d90260b3444#r28488078